renderer_cocoa/lib/Cocoa_stubs.c

```diff
@@ -256,10 +281,30 @@ CAMLprim value ml_NSWindow_setContentView(value win_v, value view_v)
   CAMLreturn(Val_unit);
 }
 
-CAMLprim value ml_NSWindow_setTitle(value win_v, value str_v)
+void ml_NSWindow_setContentView(NSWindow *win, View *view)
 {
-  CAMLparam2(win_v, str_v);
-  NSWindow *win = NSWindow_val(win_v);
+  [view setWantsLayer:YES];
+  [win setContentView:view];
+}
+
+CAMLprim value ml_NSWindow_title(NSWindow *win)
+{
+  CAMLparam0();
+  CAMLlocal1(str_v);
+
+  NSString *str = [win title];
+  const char *bytes = [str UTF8String];
+  int len = strlen(bytes);
+  str_v = caml_alloc_string(len);
+  memcpy(String_val(str_v), bytes, len);
+
+  CAMLreturn(str_v);
```

Comment:
I think for the most part we might want to keep using Obj-C strings on the OCaml side, too. After all, what's the use case of this? For the cases where you want to convert the string from NSString to OCaml's string, we could provide NSString.toString of type `nsstring => string`

Reply:
I'm not sure about this. What are the cases where you'd need to pull a string and not manipulate it at that instant?

Reply:
The use case for converting to string is when e.g you want to get a window title and change it by appending something to the end. That's pretty common practice.

In case of `NSTextInput`, use case is even more obvious. I'd want to deal with normal strings on my side and having one more step for converting from nsstring would just be a nuisance.

Reply:
@rauanmayemir your argument is convincing but in general when do you want to "get" the title? In the react model it never ever happens because you usually set things based on data. You generally don't get the title to modify it because you usually can't know what's in it. That said this function won't be used too much I suppose so I'm okay with keeping it as is.

Reply:
I'd get back to this for instances where there will be a heavy flow of strings.